forge-gui/res/cardsfolder/upcoming/the_pride_of_hull_clade.txt

```diff
@@ -0,0 +1,15 @@
+Name:The Pride of Hull Clade
+ManaCost:10 G
+Types:Legendary Creature Crocodile Elk Turtle
+PT:2/15
+K:Defender
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {X} less to cast, where X is the total toughness of creatures you control.
+SVar:X:Count$Valid Creature.YouCtrl$SumToughness
+A:AB$ Pump | Cost$ 2 U U | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | NumAtt$ 1 | SubAbility$ DBEffect | SpellDescription$ Until end of turn, target creature you control gets +1/+0, gains "Whenever this creature deals combat damage to a player, draw cards equal to its toughness," and can attack as though it didn't have defender.
+SVar:DBEffect:DB$ Effect | RememberObjects$ Targeted | Triggers$ TrigDamage | StaticAbilities$ CanAttack | ForgetOnMoved$ Battlefield
```

Comment:
it should gain the trigger

Reply:
Can the "Can Attack" static be added to the animate ability or do we need a seperate _animate_ and _effect_ ?

Reply:
that part is wrong now imo, because it's not gained